src/gc/env/gcenv.os.h

```diff
@@ -277,6 +277,12 @@ class GCToOSInterface
     // Return:
     //  Time stamp in milliseconds
     static uint32_t GetLowPrecisionTimeStamp();
+
+    // Gets the current instruction pointer.
+    // Return:
+    //  The value of the instruction pointer immediately after
+    //  the call to this function.
+    static void* GetCurrentInstructionPointer();
```

Comment:
on a related note when I look at this code, it'd be nice to remove all the code under #ifndef NO_CATCH_HANDLERS unless anyone objects...